### PR TITLE
Added puns

### PR DESCRIPTION
**Pull request by jafeltra**
This PR just adds a few puns provided by Nicole Ng and Ammu Irivinti!

### CODE REVIEW

**Comment by cmoesel** [comment]
Ugh.

**Comment by mint-thompson** [comment]
This is good and I like all of these. Thank you to our kind contributors.

**Comment by cmoesel** [comment]
I think `Ugh` is exactly the confirmation our contributors were looking for.  I hope it pleases them.

File: src/app.ts [hunk 1]
```diff
@@ -191,7 +191,8 @@ const MESSAGE_MAP: { [key in ResultStatus]: string[] } = {
     'Everything is ship-shape!',
     'Ex-clam-ation point!',
     'Ac-clam-ations!',
-    'Fin-tastic job!'
+    'Fin-tastic job!',
+    'You dolphinitely have the hang of this!'
```
Comment (ngfreiter) [inline src/app.ts:195]: Unfor-tuna-tely, I think this line is slightly too long, so it ends up looking like this:
image

It looks like it is 3 characters too long.

Comment (ngfreiter) [reply src/app.ts:195]: We could do "You're dolphinitely doing great", or "You dolphinitely got it!", or something like that. Or "You dolphinitely got the hang of it!" to stay close to the original pun.

Comment (jafeltra) [reply src/app.ts:195]: Oh good call. Somehow I like "You're dolphinitely doing great!" so I think I'll go with that!

**Pull request merged**